docs/models/openrouter.md

```diff
@@ -73,3 +73,58 @@ model = OpenRouterModel('openai/gpt-5')
 agent = Agent(model, model_settings=settings)
 ...
 ```
+
+## Image Generation
+
+You can use OpenRouter models that support image generation with the `openrouter_modalities` setting:
+
+```python {test="skip"}
+from pydantic_ai import Agent, BinaryImage
+from pydantic_ai.models.openrouter import OpenRouterModelSettings
+
+agent = Agent(
+    model='openrouter:google/gemini-2.5-flash-image-preview',
+    output_type=str | BinaryImage,
+    model_settings=OpenRouterModelSettings(openrouter_modalities=['image', 'text']),
+)
+
+result = agent.run_sync('A cat')
+assert isinstance(result.output, BinaryImage)
+```
+
+You can further customize image generation using `openrouter_image_config`:
+
+```python
+from pydantic_ai.models.openrouter import OpenRouterModelSettings
+
+settings = OpenRouterModelSettings(
+    openrouter_modalities=['image', 'text'],
+    openrouter_image_config={'aspect_ratio': '3:2'}
```

Comment:
I want this to be an option on `ImageGenerationTool` as in https://github.com/pydantic/pydantic-ai/pull/3412.